Simplenote/SPTextView.m

```diff
@@ -25,4 +27,22 @@ - (void)mouseDown:(NSEvent *)theEvent
     return [self.delegate conformsToProtocol:@protocol(SPTextViewDelegate)] ? (id<SPTextViewDelegate>)self.delegate : nil;
 }
 
+- (void)drawRect:(NSRect)dirtyRect {
+    [super drawRect:dirtyRect];
+    
+    CGFloat viewWidth = self.frame.size.width;
+    CGFloat insetX = [self shouldCalculateInset:viewWidth] ? [self getAdjustedInsetX:viewWidth] : kMinEditorPadding;
+    [self setTextContainerInset: NSMakeSize(insetX, kMinEditorPadding)];
+}
+
+- (BOOL)shouldCalculateInset: (CGFloat)viewWidth {
+    return viewWidth > kMaxEditorWidth && ![[NSUserDefaults standardUserDefaults] boolForKey:kEditorWidthPreferencesKey];
+}
+
+- (CGFloat)getAdjustedInsetX: (CGFloat)viewWidth {
+    CGFloat adjustedInset = (viewWidth - kMaxEditorWidth) / 2;
```

Comment:
We probably need to round this value. (Positions with floating values might look blurry onscreen!)